blocks/init/components/button/button.php

```diff
@@ -0,0 +1,37 @@
+<?php
+/**
+ * Template for the Button Component.
+ *
+ * @since 1.0.0
+ * @package Eightshift_Boilerplate\Blocks.
+ */
+
+namespace Eightshift_Boilerplate\Blocks;
+
+$url   = $attributes['url'] ?? '';
+$title = $attributes['title'] ?? '';
```

Comment:
Maybe rename the `title` attribute to `text` attribute.
This is usually called button text, not button title.